### ISSUE TITLE
Integrate into core?

### ISSUE THREAD

**Issue by JoshFabian** [fuzetsu/hummingbird-user-compare#3]
Hey, this is pretty great! Any interest in integrating this into the core site? I'm the founder of Hummingbird, btw ;)

**Comment by kufii** [comment]
I'd be interested in giving a shot, and I'll speak for fuzetsu in saying that I'm sure he wouldn't mind either. Do you want us to integrate into the future hummingbird?

**Comment by fuzetsu** [comment]
Yeah we'll give it a shot.

**Comment by JoshFabian** [comment]
Yep, integrating into the-future, ping me via email with the address to send invites and I'll get you both added to our Slack.
